src/main/resources/attackvectors/BlindSQLInjectionVulnerabilityPayload.properties

```diff
@@ -0,0 +1,27 @@
+BLIND_SQL_INJECTION_PAYLOAD_LEVEL_1=It allows a hacker to check if the web application is vulnerable to SQL injection. It does not show any errors. The hacker will try to inject true and false statements.<br/>\
```

Comment:
That day in the end we decided to just add payloads. can you remove this description about the Vulnerability as this is already there.
Examples given makes send but can you explain little bit ? Like 
```
As Blind sql injection doesn't show any internal details like errors or table data hence we need to find out if the injection is working or not.
Example attack to check is:
100 or 2=2 as we know 100 id is not present in the table but still it is returning true it means that the input is evaluated to true and backend has returned that car is present. 
```

Reply:
@marcin-wrotecki  anf same applies to all the other payloads too.